src/event.h

```diff
@@ -148,6 +156,7 @@ class LIBCOMMHISTORY_EXPORT Event
         ExtraProperties,
         Recipients,
         IsResolved,
+        FilterType,
```

Comment:
Leftover?

Reply:
Good catch. It's actually unused. I've removed it.